### DISCUSSION TITLE
Product packs

### DISCUSSION THREAD

**Discussion by ClementChauveauPerso** [lunarphp/lunar#1446] [Feature Requests]
It would be great to be able to offer defined product packs and have them listed with the inventory for each product.

**Comment by glennjacobs** [comment]
Would this be what some call bundles?

**Reply by woodymendoza** [reply]
Bundles / Product Packages.   I rolled out a poor mans version of this in an app once which was just a Product that referenced a list of products.  The user would adjust the price for each item and the total would be the package price. It worked for our needs. 

![CleanShot 2024-03-30 at 10 00 03](https://github.com/lunarphp/lunar/assets/879473/b6247b80-3367-4ef0-9ac6-1723dc4eb32e)

Interested in seeing how you would approach this @glennjacobs, I'm sure it will be great.
